R/use_extendr.R

```diff
@@ -114,19 +120,23 @@ use_extendr <- function(path = ".",
     dependencies = list(`extendr-api` = "*")
   )
 
-  write_file(
-    text = cargo_toml_content,
-    path = file.path("src", "rust", "Cargo.toml"),
-    search_root_from = path,
-    quiet = quiet
-  )
+  if (!file.exists(file.path("src", "rust", "Cargo.toml"))) {
```

Comment:
I see some code duplication, can we move this logic into `write_file()` or another helper function?

Reply:
Done in 548d94635799d7862b3742daea3b9aef23418f45